naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/PersistentConsistencyServiceDelegateImpl.java

```diff
@@ -86,6 +87,9 @@ public boolean isAvailable() {
     }
     
     private PersistentConsistencyService switchOne() {
+        if (ApplicationUtils.getStandaloneMode()) {
+            return oldPersistentConsistencyService;
```

Comment:
If standalone, should we use new consistency service?
And can we make switchNewPersistentService = true rather than directly return oldPersistentConsistencyService

Reply:
If it is in this form, then the purpose of replacing the old raft before does not exist, this should be optimized in the new PersistentService